Source/BoundaryConditions/PML.cpp

```diff
@@ -586,17 +583,15 @@ PML::PML (const int lev, const BoxArray& grid_ba, const DistributionMapping& /*g
         // assuming that the bounding box around grid_cba is a single patch, and not disjoint patches, similar to fine patch.
         amrex::Box domain1 = grid_cba.minimalBox();
         for (int idim = 0; idim < AMREX_SPACEDIM; ++idim) {
-            if ( ! cgeom->isPeriodic(idim)) {
-                if (do_pml_Lo[idim]){
-                    // ncell is divided by refinement ratio to ensure that the
-                    // physical width of the PML region is equal is fine and coarse patch
-                    domain1.growLo(idim, -ncell/ref_ratio[idim]);
-                }
-                if (do_pml_Hi[idim]){
-                    // ncell is divided by refinement ratio to ensure that the
-                    // physical width of the PML region is equal is fine and coarse patch
-                    domain1.growHi(idim, -ncell/ref_ratio[idim]);
-                }
+            if (do_pml_Lo[idim]){
+                // ncell is divided by refinement ratio to ensure that the
+                // physical width of the PML region is equal is fine and coarse patch
```

Comment:
This comment was already there before your PR, but since you are modifying these lines anyways, would you mind fixing it as well?
```suggestion
                // physical width of the PML region is equal in fine and coarse patch
```